test/io-tests/test_io.py

```diff
@@ -157,7 +164,7 @@ def test_expected_config(expectedconfig):
     assert dumpconfig(basedir / "configs" / expectedconfig) == expected
 
 
-def test_stable_config(configpath):
+def test_stable_config(tmp_path, configpath):
```

Comment:
this is the first argument that is snake-cased. all others are just like `tmppath`. it seems like your idea was to only snake-case the test function names?

Reply:
but tbh: I don't really get where it actually comes from? Is this a pytest built-in fixture or something?

Reply:
that's a pytest builtin fixture, have to use that name... I'll put some of the longer names in snake case later, too